testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -206,7 +207,8 @@ def params_from_base_suite_setup(request):
             cluster_config = "{}/base_lb_{}".format(CLUSTER_CONFIGS_DIR, mode)
 
     log_info("Using '{}' config!".format(cluster_config))
-    cluster_utils = ClusterKeywords(cluster_config)
+    cluster_utils = ClusterKeywords()
```

Comment:
This change has to be reverted to `cluster_utils = ClusterKeywords(cluster_config)`

Reply:
done